http.go

```diff
@@ -98,6 +102,10 @@ func InstrumentHandler(label string, handler http.Handler) http.Handler {
 		}
 		httpRequestBytesReturned.Get(labels).Put(float64(writer.responseSize(r)))
 	})
+	const traceSampleRate = 0.01
```

Comment:
I think we had discussed potentially doing time-based sampling. I.e., sample if X seconds have elapsed since the last sample. One benefit of that is that even low frequency services will get a decent set of samples while heavier ones will be sampled more aggressively.

Maybe once a second or once every five or ten seconds?

Reply:
Thanks, this is much better indeed. Set it to once every second for now.